server/webdriver/base_utils.py

```diff
@@ -25,7 +25,7 @@
 def create_driver(preferences=None):
   # These options are needed to run ChromeDriver inside a Docker without a UI.
   chrome_options = Options()
-  chrome_options.add_argument('--headless=new')
+  # chrome_options.add_argument('--headless=new')
```

Comment:
Was this intentional? Should we just delete the line entirely?

Reply:
Reverted! 
Definitely should keep it because without it, it ends up showing you all the tests that run which is great for debugging but insanely annoying when it keeps taking over your cursor :)